src/build/service.js

```diff
@@ -43,8 +43,6 @@ class BuildService {
       AutoRemove: true,
       User: `${os.userInfo().uid}:${os.userInfo().gid}`,
       Binds: [
-        '/etc/passwd:/etc/passwd:ro',
-        '/etc/group:/etc/group:ro',
```

Comment:
I don't think this will work - stuff that expects a username will fail. But I'll try something similar.